pkg/operator/defaults.go

```diff
@@ -48,35 +48,9 @@ var (
 	DefaultPrometheusConfigReloaderImage = "quay.io/prometheus-operator/prometheus-config-reloader:v" + version.Version
 
 	// PrometheusCompatibilityMatrix is a list of supported prometheus versions.
-	// prometheus-operator end-to-end tests verify that the operator can deploy from LTS n-1 to the latest stable.
+	// prometheus-operator end-to-end tests verify that the operator can deploy from the current LTS version to the latest stable release.
 	// This list should be updated every time a new LTS is released.
 	PrometheusCompatibilityMatrix = []string{
```

Comment:
If we remove these versions from here, we won't be set these versions in CR?

Reply:
No, the versions here are only used for tests (and documentation).